static/js/tools/scatter2/place_options.tsx

```diff
@@ -88,31 +88,31 @@ function PlaceOptions(): JSX.Element {
   }, [place.value]);
 
   return (
-    <Card>
+    <Card className="place-options-card">
       <Container className="place-options">
-        <Row className="centered-items-row">
-          <Col xs="auto">Plot places in</Col>
-          <Col>
-            <div id="search">
-              <SearchBar
-                places={
-                  place.value.enclosingPlace.dcid
-                    ? {
-                        [place.value.enclosingPlace.dcid]:
-                          place.value.enclosingPlace.name,
-                      }
-                    : {}
-                }
-                addPlace={(e) => selectEnclosingPlace(place, e)}
-                removePlace={() =>
-                  unselectEnclosingPlace(place, setChildPlaceTypes)
-                }
-                numPlacesLimit={1}
-              />
-            </div>
-          </Col>
-          <Col xs="auto">of type</Col>
-          <Col xs="3">
+        <div className="place-options-section place-search-section">
```

Comment:
feel free to use id's too - in this case, place-search-section and place-type-section are probably going to be unique identifiers

Reply:
done